std/uni.d

```diff
@@ -7478,7 +7478,7 @@ public:
 
         See_Also: $(LREF Grapheme.valid)
     +/
-    ref opOpAssign(string op)(dchar ch)
+    ref opOpAssign(string op)(dchar ch) @trusted
     {
         static if (op == "~")
         {
```

Comment:
It we can use stdx.checked in general - it's just not @safe by default atm, but there's https://github.com/dlang/phobos/pull/5928